src/pczt/parse.rs

```diff
@@ -54,6 +54,17 @@ impl<D: OrchardDomainCommon> Bundle<D> {
             .into_option()
             .ok_or(ParseError::InvalidAnchor)?;
 
+        let burn = burn
+            .into_iter()
+            .map(|(burn_asset, burn_value)| {
+                let burn_asset = AssetBase::from_bytes(&burn_asset)
+                    .into_option()
+                    .ok_or(ParseError::InvalidAnchor)?;
```

Comment:
need new error. `InvalidAnchor` is not relevant

Reply:
Done